apps/heft/src/cli/actions/CustomAction.ts

```diff
@@ -31,6 +31,7 @@ export interface ICustomActionParameterStringList extends ICustomActionParameter
 }
 
 /** @beta */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 export interface ICustomActionParameterBase<TParameter extends CustomActionParameterType> {
```

Comment:
@iclanton What's going on here? Why do we have a interface with a generic parameter that is unused?